### DISCUSSION TITLE
Integrate Grocy shopping / chores/ tasks/ calendar with HA equivalents

### DISCUSSION THREAD

**Discussion by Yowaishi** [hassio-addons/addon-grocy#448] [Ideas]
Prior to Home Assistant implementing tasks / calendars people had a variety of ideas for custom grocy cards / components in Home Assistant ([see here for an example](https://community.home-assistant.io/t/grocy-custom-component-and-card-s/218978?page=9)). Examples included checking off tasks(chores), and integrating the calendar (for example, is the recycling supposed to be out today shows on a calendar as well as as a chore/task).

Now that Home Assistant supports tasks/calendars I was wondering if there were plans to make it easier for a new user to integrate grocy with those native HA components? I tried doing google searches across forums (here on github, reddit, HA community) as well to see if people were thinking about this but didn't see anything that seemed to be what I'm suggesting. Apologies if it's already been suggested.

For example 

- there could be a calendar in HA called 'grocy' that shows when chores are to be done
- A todo list in HA that pulls items from Grocy's chores that are due/overdue
- a shopping list that you can add items to using Home Assistant's to do list (and your voice) that also lists the items from Grocy's curated shopping list.

**Comment by frenck** [comment]
I think you are mixing up integrations vs add-ons, as this repository is an add-on, which only runs an application/service next to Home Assistant. In no way, add-ons adds stuff to Home Assistant itself. Integrations do that; hence, you are in the wrong place.

../Frenck

**Comment by Yowaishi** [comment]
thank you kindly for pointing out that I am in the wrong place. Can you kindly direct me to the grocy integration page where I can submit the idea?

**Reply by frenck** [reply]
I have no idea what or where that is. I am not a user of that myself.
